# HG changeset patch
# User Michael Goffioul <michael.goffioul@gmail.com>
# Date 1390102987 18000
#      Sat Jan 18 22:43:07 2014 -0500
# Node ID c1baf94184af9d3ded5664e454ade931964c34cb
# Parent  a97ed15d1ea8fd235a37ec828bcdf600a00f2e38
* oct-parse.in.yy: Suppress TAB characters.

diff --git a/libinterp/parse-tree/oct-parse.in.yy b/libinterp/parse-tree/oct-parse.in.yy
--- a/libinterp/parse-tree/oct-parse.in.yy
+++ b/libinterp/parse-tree/oct-parse.in.yy
@@ -1623,35 +1623,35 @@ methods_block   : METHODS stash_comment 
                       {
                         // make_classdef_methods_block deleted $3 and $5.
                         ABORT_PARSE;
                       }
                   }
                 ;
 
 method_decl1    : identifier
-		  {
+                  {
                     if (! ($$ = parser.start_classdef_external_method ($1, 0)))
                       ABORT_PARSE;
                   }
-		| identifier param_list
-		  { if (! ($$ = parser.start_classdef_external_method ($1, $2)))
+                | identifier param_list
+                  { if (! ($$ = parser.start_classdef_external_method ($1, $2)))
                       ABORT_PARSE;
                   }
                 ;
 
 method_decl     : stash_comment method_decl1
-		  { $$ = parser.finish_classdef_external_method ($2, 0, $1); }
+                  { $$ = parser.finish_classdef_external_method ($2, 0, $1); }
                 | stash_comment return_list '=' method_decl1
-		  { $$ = parser.finish_classdef_external_method ($4, $2, $1); }
+                  { $$ = parser.finish_classdef_external_method ($4, $2, $1); }
                 ;
 
 method          : method_decl
-		  { $$ = $1; }
-		| function
+                  { $$ = $1; }
+                | function
                   { $$ = $1; }
                 ;
 
 methods_list    : method
                   {
                     octave_value fcn;
                     if ($1)
                       fcn = $1->function ();
@@ -3241,17 +3241,17 @@ octave_base_parser::make_classdef_enum_b
       delete elist;
     }
 
   return retval;
 }
 
 octave_user_function*
 octave_base_parser::start_classdef_external_method (tree_identifier *id,
-		                                    tree_parameter_list *pl)
+                                                    tree_parameter_list *pl)
 {
   octave_user_function* retval = 0;
 
   // External methods are only allowed within @-folders. In this case,
   // curr_class_name will be non-empty.
 
   if (! curr_class_name.empty ())
     {
@@ -3290,17 +3290,17 @@ octave_base_parser::start_classdef_exter
   if (! retval)
     delete id;
 
   return retval;
 }
 
 tree_function_def *
 octave_base_parser::finish_classdef_external_method (octave_user_function *fcn,
-		                                     tree_parameter_list *ret_list,
+                                                     tree_parameter_list *ret_list,
                                                      octave_comment_list *cl)
 {
   if (ret_list)
     fcn->define_ret_list (ret_list);
 
   if (cl)
     fcn->stash_leading_comment (cl);
 
